lib/periodicChecks.js

```diff
@@ -154,7 +154,7 @@ const ensureAllIssuesHaveProjects = async (context) => {
 
   const issuesWithoutProjects = allOpenIssues.filter((issue) => {
     const issueBelongsToProject = allProjectCards.some((card) => {
-      return card.content_url.endsWith(issue.number);
+      return card.content_url && card.content_url.endsWith(issue.number);
```

Comment:
Why can `content_url` be undefined?

Reply:
When a project card hasn't been linked to an issue. For example, if the project card just contains notes.